### PR TITLE
Configure master and node --loglevel option from the CLI

### PR DESCRIPTION
**Pull request by EricMountain-1A**
Allows to configure master and node --loglevel option from the command line.

### CODE REVIEW

**Comment by jaryn** [comment]
Can one of the admins verify this patch?

**Comment by jaryn** [comment]
Can one of the admins verify this patch?

**Comment by sdodson** [comment]
I've verified that this doesn't break anything for byo playbooks, however it looks like byo playbooks weren't working properly here so I'll need to revisit that.

LGTM based on the fact that it doesn't break byo playbooks.

**Comment by arilivigni** [comment]
Testing job has been triggered.

**Comment by arilivigni** [comment]
Internal Jenkins error during triggering.
 No test results found.

**Comment by detiber** [comment]
@EricMountain-1A can you rebase this to resolve the conflict?

Otherwise, :+1: from me

**Comment by EricMountain-1A** [comment]
Thanks, @detiber. Rebase done.

File: roles/openshift_master/tasks/main.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@
       cluster_method: "{{ openshift_master_cluster_method | default(None) }}"
       cluster_hostname: "{{ openshift_master_cluster_hostname | default(None) }}"
       cluster_public_hostname: "{{ openshift_master_cluster_public_hostname | default(None) }}"
-      debug_level: "{{ openshift_master_debug_level | default(openshift.common.debug_level) }}"
+      debug_level: "{{ lookup('oo_option', 'openshift_master_debug_level') | default(openshift.common.debug_level, True) }}"
```
Comment (detiber) [inline roles/openshift_master/tasks/main.yml]: This should remain using openshift_master_debug_level.

The lookups in the playbooks/*/openshift-cluster/config.yml playbooks are already setting openshift_master_debug_level to the value from the lookup, and we don't really want the roles to have dependencies on environment variables or cli driven variables.

Comment (detiber) [inline roles/openshift_master/tasks/main.yml]: Ah, please ignore my comment above about already setting openshift_master_debug level...  instead we should be setting openshift_master_debug_level in the playbooks, and keeping the role using the openshift_master_debug_level variable.

File: roles/openshift_node/tasks/main.yml [hunk 2]
```diff
@@ -23,7 +23,7 @@
   - role: node
     local_facts:
       annotations: "{{ openshift_node_annotations | default(none) }}"
-      debug_level: "{{ openshift_node_debug_level | default(openshift.common.debug_level) }}"
+      debug_level: "{{ lookup('oo_option', 'openshift_node_debug_level') | default(openshift.common.debug_level, True) }}"
```
Comment (detiber) [inline roles/openshift_node/tasks/main.yml]: Same comments as above about openshift_master_debug level. I don't want to extend the cli/env lookups into the roles if at all possible.

**Comment by detiber** [comment]
@EricMountain-1A Thanks for rebasing, I just have a couple of concerns related to extending the lookups into the roles, and would like to see those moved out into the playbooks instead.

**Pull request closed without merging**